dotnet/src/Experimental/Process.Runtime.Dapr/Actors/ProcessActor.cs

```diff
@@ -308,8 +308,7 @@ private async Task Internal_ExecuteAsync(Kernel? kernel = null, int maxSuperstep
                 await Task.WhenAll(stepProcessingTasks).ConfigureAwait(false);
 
                 // Handle public events that need to be bubbled out of the process.
-                var eventQueue = this.ProxyFactory.CreateActorProxy<IEventBuffer>(new ActorId(this.Id.GetId()), nameof(EventBufferActor));
```

Comment:
Can we remove `localKernel` and `messageChannel` on lines 274/275? I don't see them being used.

Reply:
Yes, good catch